tests/test_codebase/test_mmdet/test_mmdet_models.py

```diff
@@ -2364,3 +2364,251 @@ def test_solov2_head_predict_by_feat(backend_type):
                 atol=1e-05)
     else:
         assert rewrite_outputs is not None
+
+
+def get_condinst_bbox_head():
+    """condinst Bbox Head Config."""
+    test_cfg = Config(
+        dict(
+            mask_thr=0.5,
+            max_per_img=100,
+            min_bbox_size=0,
+            nms=dict(iou_threshold=0.6, type='nms'),
+            nms_pre=1000,
+            score_thr=0.05))
+    from mmdet.models.dense_heads import CondInstBboxHead
+    model = CondInstBboxHead(
+        center_sampling=True,
+        centerness_on_reg=True,
+        conv_bias=True,
+        dcn_on_last_conv=False,
+        feat_channels=256,
+        in_channels=256,
+        loss_bbox=dict(loss_weight=1.0, type='GIoULoss'),
+        loss_centerness=dict(
+            loss_weight=1.0, type='CrossEntropyLoss', use_sigmoid=True),
+        loss_cls=dict(
+            alpha=0.25,
+            gamma=2.0,
+            loss_weight=1.0,
+            type='FocalLoss',
+            use_sigmoid=True),
+        norm_on_bbox=True,
+        num_classes=80,
+        num_params=169,
+        stacked_convs=4,
+        strides=[
+            8,
+            16,
+            32,
+            64,
+            128,
+        ],
+        test_cfg=test_cfg,
+    )
+
+    model.requires_grad_(False)
+    return model
+
+
+@pytest.mark.skipif(
+    reason='Only support GPU test', condition=not torch.cuda.is_available())
+@pytest.mark.parametrize('backend_type',
+                         [Backend.ONNXRUNTIME, Backend.TENSORRT])
+def test_condinst_bbox_head_predict_by_feat(backend_type):
+    """Test predict_by_feat rewrite of condinst bbox head."""
+    check_backend(backend_type)
+    condinst_bbox_head = get_condinst_bbox_head()
+    condinst_bbox_head.cpu().eval()
+    s = 128
+    batch_img_metas = [{
+        'scale_factor': np.ones(4),
+        'pad_shape': (s, s, 3),
+        'img_shape': (s, s, 3)
+    }]
+
+    output_names = ['dets', 'labels', 'param_preds', 'points', 'strides']
+    deploy_cfg = Config(
+        dict(
+            backend_config=dict(
+                type=backend_type.value,
+                common_config=dict(max_workspace_size=1 << 32),
+                model_inputs=[
```

Comment:
could remove these input shapes setting and they are not correct for inputs like:`cls_scores, bbox_preds, label, ...`